src/Form/LinkBlockFormDataProvider.php

```diff
@@ -127,6 +135,43 @@ public function getData()
         ]];
     }
 
+    /**
+     * Make sure to fill empty multilang fields if value for default is available
+     * 
+     * @param array $linkBlock
+     * 
+     * @return array
+     */
+    public function prepareData(array $linkBlock): array
+    {
+        $defaultLanguageId = (int) $this->configuration->get('PS_LANG_DEFAULT');
+
+        if (!empty($linkBlock['block_name'])) {
+            foreach ($this->languages as $language) {
+                if (empty($linkBlock['block_name'][$language['id_lang']])) {
+                    $linkBlock['block_name'][$language['id_lang']] = $linkBlock['block_name'][$defaultLanguageId];
+                }
+            }
+        }
+
+        if (!empty($linkBlock['custom'])) {
+            foreach ($linkBlock['custom'] as $key => $customLanguages) {
+                if ($this->isEmptyCustom($customLanguages)) {
+                    continue;
+                }
+
+                foreach ($customLanguages as $idLang => $custom) {
+                    $linkBlock['custom'][$key][$idLang] = [
+                        'title' => !empty($custom['title']) ? $custom['title'] : $customLanguages[$defaultLanguageId]['title'],
```

Comment:
```suggestion
                        'title' => $custom['title'] ?? $customLanguages[$defaultLanguageId]['title'],
```
Should be enough?

Reply:
Thank you, you're right, `empty` tricked me